static/coffee/simulator.coffee

```diff
@@ -42,6 +42,17 @@ window.updateSimulator = (data) ->
     media_type = null
     media_viewer_elt = null
 
+    quick_replies = null
+
+    if msg.metadata and Object.keys(msg.metadata).length > 1
```

Comment:
This should be checking for the quick replies key, not on the total length of metadata which can contain who knows what.